src/adapters/mc/asb/CMakeLists.txt

```diff
@@ -23,15 +23,15 @@ set_target_properties(OsConfigResourceAsb PROPERTIES OUTPUT_NAME libOsConfigReso
 
 # Create the policy artifacts ZIP package for the Azure Security Baseline for Linux
 
-add_custom_target(stage_create_zip
+add_custom_target(stage_create_zip_asb
     COMMAND ${CMAKE_COMMAND} -E make_directory ${PROJECT_BINARY_DIR}/Staging
```

Comment:
Do we also need mc/ssh one to be changed or only this one to make the two different?

Reply:
just this one

Reply:
Let's also rename that one state_create_ssh_zip to keep consistent. Thanks